index.html

```diff
@@ -28,89 +28,199 @@
         company: "Akamai",
         companyURL: "https://akamai.com/"
       }
-      // Add additional editors here.
-      // https://github.com/w3c/respec/wiki/editors
-    ],
-    "shortName": "priority-hints",
-    "specStatus": "CG-DRAFT",
-    "subjectPrefix": "[priority-hints]",
-    "wg": "WICG",
-    "otherLinks": [{
-      "key": "Repository",
-      "data": [{
-        "value": "We are on GitHub.",
-        "href": "https://github.com/WICG/priority-hints"
-      }, {
-        "value": "File a bug.",
-        "href": "https://github.com/WICG/priority-hints/issues"
-      }, {
-        "value": "Commit history.",
-        "href": "https://github.com/WICG/priority-hints/commits/gh-pages"
-      }, ]
-    }],
-  };
+        // Add additional editors here.
+        // https://github.com/w3c/respec/wiki/editors
+      ],
+      "shortName": "priority-hints",
+      "specStatus": "CG-DRAFT",
+      "subjectPrefix": "[priority-hints]",
+      "wg": "WICG",
+      "otherLinks": [{
+        "key": "Repository",
+        "data": [{
+          "value": "We are on GitHub.",
+          "href": "https://github.com/WICG/priority-hints"
+        }, {
+          "value": "File a bug.",
+          "href": "https://github.com/WICG/priority-hints/issues"
+        }, {
+          "value": "Commit history.",
+          "href": "https://github.com/WICG/priority-hints/commits/gh-pages"
```

Comment:
This link doesn't work which leads me to think we're serving GH pages off of master?

Reply:
Well spotted! I've updated the link in the latest commit.